holidays/countries/spain.py

```diff
@@ -125,10 +132,16 @@ def _populate(self, year):
             "VC",
         }:
             self[easter(year) + rd(days=+1)] = "Lunes de Pascua"
-
-        if 2022 != year:
-            self._is_observed(date(year, MAY, 1), "Día del Trabajador")
-        elif 2022 == year and self.subdiv in {
+        elif year >= 2022 and self.subdiv in {
+            "RI",
```

Comment:
Could you sort these for consistency with rest of the code?

Reply:
Done